CMakeLists.txt

```diff
@@ -703,6 +703,12 @@ if(LIBUV_BUILD_TESTS)
     set_tests_properties(uv_test PROPERTIES ENVIRONMENT
                          "LIBPATH=${CMAKE_BINARY_DIR}:$ENV{LIBPATH}")
   endif()
+  if(WIN32)
+    add_custom_command(TARGET uv_run_tests POST_BUILD
+                       COMMAND "${CMAKE_COMMAND}" -E copy
+                       "$<TARGET_FILE:uv_run_tests>"
+                       "$<TARGET_FILE_DIR:uv_run_tests>/uv_run_tests_no_ext")
```

Comment:
These will also need to go into the autotools build recipe too

Reply:
This is only relevant on Windows. Does the autotools recipe work there?

Reply:
I also know very little about autotools, how would I go about this?

Reply:
Please see my changes to `Makefile.am`. I'm an autotools newbie, not sure if I did it right.

Reply:
I don't know autotools well either, so that seems as good a guess as any about how to do that